runner/src/scenario/ceramic/model_instance.rs

```diff
@@ -73,6 +73,7 @@ pub struct EnvBasedConfig {
 
 #[derive(Clone, Debug)]
 pub struct GooseUserInfo {
+    #[allow(dead_code)]
```

Comment:
rust updated on CI and figured out that these variables were not being read. @dav1do is that intentional are these present for future use?

Reply:
That is really weird.. both of these are used by scenarios currently... maybe we can pair quickly to see what's up? I'm not seeing any clippy lints locally so I'm a bit confused. This is the only thing I'd like to sort out before merging, the other comments are just observations.